x11-plugins/wmpager/wmpager-1.2-r2.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A simple pager docklet for the WindowMaker window manager"
+HOMEPAGE="http://wmpager.sourceforge.net/"
+SRC_URI="mirror://sourceforge/wmpager/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~sparc ~x86"
+IUSE=""
```

Comment:
Empty variable can be dropped.

Reply:
@hydrapolic every thrid ebuild has it empty, so nobody just caries (despite I agree with you it can be dropped)

Reply:
Once we learn it we don't put the empty variables there again.